core/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/status/SnapshotsStatusRequest.java

```diff
@@ -112,17 +114,40 @@ public SnapshotsStatusRequest snapshots(String[] snapshots) {
         return this;
     }
 
+    /**
+     * Set to <code>true</code> to ignore unavailable snapshots, instead of throwing an exception.
+     * Defaults to <code>false</code>, which means unavailable snapshots cause an exception to be thrown.
+     *
+     * @param ignoreUnavailable whether to ignore unavailable snapshots
```

Comment:
I don't think the `@param` or `@return` adds anything to the information. Personally I wouldn't add them. You can do what you want. I just wanted to let you know its ok (by me) not to add them if they don't make the API more clear.